### ISSUE TITLE
Accessibility standards

### ISSUE THREAD

**Issue by ZachSAEON** [SAEONData/catalogue#14]
Since we are using the Material UI component library, which so far as I know fully implements 'accessibility', the atlas should be fully accessible to.
